generators/ci-cd/templates/.gitlab-ci.yml.ejs

```diff
@@ -152,18 +152,20 @@ before_script:
     <%_ } _%>
     - export MAVEN_USER_HOME=`pwd`/.maven
     <%_ if (!skipClient) { _%>
-        <%_ if (clientPackageManager === 'yarn') { _%>
+        <%_ if (!insideDocker) { _%>
```

Comment:
We could do something similar for the gradle build too. I think.

Reply:
For sure it should be possible. I always want to try gradle, I guess now it will be a good point 😃.